ported/libibverbs/rc_pingpong.c

```diff
@@ -508,11 +508,12 @@ int main(int argc, char *argv[])
 	hints->ep_attr->type = FI_EP_MSG;
 	hints->caps = FI_MSG;
 	hints->mode = FI_LOCAL_MR;
-	hints->addr_format = FI_SOCKADDR;
 
 	rc = ft_read_addr_opts(&node, &service, hints, &flags, &opts);
-	if (rc)
-		return 1;
+	if (rc) {
+		FT_PRINTERR("ft_read_addr_opts", rc);
+		return -rc;
```

Comment:
Other places simply return the result of ft_read_addr_opts, but here we negate it.